Display icon/image on fedify lookup (#348)
fedify-dev/fedify: CHANGES.md

 -  The `fedify` CLI now correctly disables color output when standard output
    isn't a TTY (for example, when redirecting to a file) or when the `NO_COLOR`
    environment variable is set.  [[#257], [#341] by Cho Hasang]
    

>  -  The 'fedify lookup' command now displays images depending on user's 

Reviewer: ```suggestion
 -  The `fedify lookup` command now displays images depending on user's 
```

    terminal emulator. [[#169], [#348] by Jiwon Kwon]

>      - Supported terminal emulators are Kitty, WezTerm, Konsole, Warp,
>       Wayst, st, and iterms.

Reviewer: Alignment!

```suggestion
     -  Supported terminal emulators are Kitty, WezTerm, Konsole, Warp,
        Wayst, st, and iTerm.
```

>      - Supported terminal emulators are `Kitty`, `WezTerm`, `Konsole`, `Warp`,
>       `Wayst`, `st`, and `iterms`.

Reviewer: ```suggestion
     -  Supported terminal emulators are Kitty, WezTerm, Konsole, Warp,
        Wayst, st, and iTerm.
```
